cocos/math/CCGeometry.cpp

```diff
@@ -90,8 +91,8 @@ void Size::setSize(float w, float h)
 
 bool Size::equals(const Size& target) const
 {
-    return (fabs(this->width  - target.width)  < FLT_EPSILON)
-        && (fabs(this->height - target.height) < FLT_EPSILON);
+    return (std::abs(this->width  - target.width)  < FLT_EPSILON)
+        && (std::abs(this->height - target.height) < FLT_EPSILON);
```

Comment:
[the std::abs doc](http://en.cppreference.com/w/cpp/numeric/math/abs) says it accepts integer?